Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM ubuntu:16.04
+
+MAINTAINER Jake Bunce <jake@omise.co>
+
+ARG USER=plasma
+ARG GROUP=plasma
+ARG UID=1000
+ARG GID=1000
+ARG HOME=/home/$USER
+
+RUN groupadd --gid "${GID}" "${USER}" && \
+    useradd \
+      --uid ${UID} \
+      --gid ${GID} \
+      --create-home \
+      --shell /bin/bash \
+      ${USER}
+
+ARG BUILD_PACKAGES="build-essential autoconf libtool libgmp3-dev libssl-dev wget gettext"
```

Comment:
libgmp3-dev -> libgmp-dev. We need to update the readme.

Reply:
ack